vault/blog.2021-06-24-seed-bank.md

```diff
@@ -0,0 +1,69 @@
+---
+id: qTeL51LFD0Y8uC9ect7QV
+title: The Dendron Seed Bank 
+desc: ''
+updated: 1624569852953
+created: 1624562569744
+---
+
+![seed bank](https://org-dendron-public-assets.s3.amazonaws.com/images/markus-spiske-4PG6wLlVag4-unsplash.jpg)
+
+
+## Summary
+Dendron is the note-taking tool that helps **people** manage **any amount of information**. 
+
+Today, we are announcing the preview of the Seed Bank - a public registry for general knowledge. 
+
+If you're familiar with package managers like npm, it's kind of like that but for any sort of information, not just code.
+
+The seed bank makes it easy to **discover, reference, and grow** information on things that you care about. This could be a guide on [personal knowledge management](http://pkm.dendron.so/) or a reference to everything [AWS](https://aws.dendron.so/)
+
+With the seed bank, you no longer need to start from zero when looking stuff up - rather, you have a single source of reference that you can grow over time.
+
+## Use Case
+
+This is best demonstrated with an example. You are Jenny, a DevOps engineer that is responsible for managing all deployments, ETL pipelines, and general *backend-ish things* at a recently funded tech startup. 
+
+You live in the shell and dream in POSIX but even you don't remember all the arcane incantations to command the shell in all situations.
+
+![I don't know what's worse--the fact that after 15 years of using tar I still can't keep the flags straight, or that after 15 years of technological advancement I'm still mucking with tar flags that were 15 years old when I started.](https://imgs.xkcd.com/comics/tar.png)
+> "[tar](https://xkcd.com/1168/)", by Randall Munroe, licensed under Creative Commons Attribution-NonCommercial 2.5 License
+
+You already use Dendron as a daily journal to track your tasks and you are tired of using google to find the same `curl` option to `POST` data that you looked up at least a dozen times before. You take a visit to the seed bank. 
+
+Because the seed bank is currently in preview, you open up a shell and navigate to your workspace. Since you already have dendron open in vscode, you run `> Create new Integrated Terminal` to open up a shell. 
```

Comment:
Since you've already established Jenny's comfortable with shell, might make sense just to drop the "Because seed bank is in preview ...". Turns a bug into a feature.

Reply:
haha, good point. updated!